internal/protocol/guestresource/resources.go

```diff
@@ -66,7 +66,7 @@ type WCOWCombinedLayers struct {
 type LCOWMappedVirtualDisk struct {
 	MountPath  string            `json:"MountPath,omitempty"`
 	Lun        uint8             `json:"Lun,omitempty"`
-	Controller uint8             `json:"Controller,omitempty"`
+	Controller string            `json:"Controller,omitempty"`
```

Comment:
Owww very interesting solution! Do we care about breaking changes here? Older shims wont work

Reply:
I agree we need to be mindful about breaking changes in the bridge protocol. Maybe it's okay here, but could be a problem in the future. Maybe add the new field as `ControllerStr`, and check which field is set?

Reply:
Alternately, if we implement the string -> GUID conversion logic ourselves, we don't need to change the protocol at all.

Reply:
@kevpar Do you mean that we should maintain the list of GUIDs in opengcs too and then just lookup the GUID from the controller number passed?

Reply:
That's definitely an option. Though what I was actually referring to above was we could duplicate the logic HCS uses to convert a controller string ID to a GUID, and then use it ourselves in the gcs.

Reply:
I see. I like the option of keeping the same GUID list in opengcs because it doesn't break the protocol but at the same time makes it very clear that we are referring to each controller with a hardcoded GUID rather than depending on hcs to do this controller number to GUID conversion. I will make changes to follow that logic. Let me know if you think it's not a good idea.

Reply:
I think that's fine. I doubt HCS will change the GUID generation logic, but if they do this way we'll keep working.